demo/package.json

```diff
@@ -4,22 +4,17 @@
     "description": "Express app demonstrating usage of Traffic Cop.",
     "main": "index.js",
     "scripts": {
```

Comment:
**question:** Were the webpack changes intentional? I wasn't able to get it to build from the demo folder.

Reply:
Ah sorry, no. I forgot to include in the testing instructions that you need to run `npm install && npm run build` first in the root directory before starting up the demo 😶‍🌫️ 

Removing webpack from the demo should be fine here, as all it was doing before this change was importing the cookie helper dependency.